specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2016-06-01/vaults.json

```diff
@@ -928,6 +1007,218 @@
         },
         "upgradeDetails": {
           "$ref": "#/definitions/UpgradeDetails"
+        },
+        "privateEndpointConnections": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnectionVaultProperties"
+          },
+          "description": "List of private endpoint connection.",
+          "readOnly": true
+        },
+        "privateEndpointStateForBackup": {
+          "description": "Private endpoint state for backup.",
+          "enum": [
+            "None",
+            "Enabled"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "VaultPrivateEndpointState",
+            "modelAsString": true
+          }
+        },
+        "privateEndpointStateForSiteRecovery": {
+          "description": "Private endpoint state for site recovery.",
+          "enum": [
+            "None",
+            "Enabled"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "VaultPrivateEndpointState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "IdentityData": {
+      "required": [
+        "type"
+      ],
+      "type": "object",
+      "description": "Identity for the resource.",
+      "properties": {
+        "principalId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The principal ID of resource identity."
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The tenant ID of resource."
+        },
+        "type": {
+          "type": "string",
+          "description": "The identity type.",
+          "enum": [
+            "SystemAssigned",
+            "None"
+          ],
+          "x-ms-enum": {
+            "name": "ResourceIdentityType",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "PrivateEndpointConnectionVaultProperties": {
+      "description": "Information to be stored in Vault properties as an element of privateEndpointConnections List. It is to be returned to RRP from BMS as response of PUT PECP and PUT PEC calls.",
```

Comment:
>It is to be returned to RRP from BMS as response of PUT PECP and PUT PEC calls [](start = 117, length = 78)

these acronyms don't mean anything to me and won't to the customer either

Reply:
Removed this line from description